index.html

```diff
@@ -37,5 +37,161 @@ <h3>LET'S CONNECT</h3>
     </div>
 
 
+    <section id="portfolio" class="card">
+        <div class="section-odd">
+          <div class="case">
+            <div class="first-section-background">
+              <div class="image">
+                <img src="image/Snapshoot Portfolio.png" alt="The Tonic image">
+              </div>
+              <div class="content">
+                <h2>Tonic</h2>
+                <ul>
+                    <li class="canopy">CANOPY</li>
+                    <li><img src="image/Counter.png" alt=""></li>
+                    <li>Back End Dev</li>
+                    <li><img src="image/Counter.png" alt=""></li>
+                    <li>2015</li>
+                </ul>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul>
+                    <li><a href="">html</a></li>
+                    <li><a href="">css</a></li>
+                    <li><a href="">javaScript</a></li>
+                </ul>
+                <button type="button"><a href="">See Project</a></button>
+              </div>
+            </div>
+          </div>
+        </div>
+  
```

Comment:
- [ ] Please 🙏🏽 follow the mobile design for this milestone otherwise your design will have some errors in the mobile version take in mind there are representative differences between the desktop version and the mobile version, for now, pay attention to the details in the mobile version, kindly correct the errors below to be similar to the Figma design 👇🏽 

- The corner and the extension of the home page:

| Figma design | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/212328339-3433a525-7c11-4f04-9a18-9c0d91ad0a6d.png) | ![image](https://user-images.githubusercontent.com/58642949/212328256-14fa531c-7a6d-4165-86ca-bc1fbb0e261b.png) |

- The image in the first project 

| Figma design | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/212328695-53cd55a8-ace1-495a-8833-28044c2c88f3.png) | ![image](https://user-images.githubusercontent.com/58642949/212328879-2e5f9a17-9afc-45fe-a361-797ead8fcd02.png) |

- The image and coding languages in the second project

| Figma design | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/212329302-b957ef87-e080-4a17-ab5d-fe7d5b368167.png) | ![image](https://user-images.githubusercontent.com/58642949/212329171-c8a14c9a-b701-4788-9ab7-9b4e81f287fa.png) |

- The image, title and coding languages in the third project

| Figma design | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/212329765-cae63c49-3dd7-44e0-bb60-00ba44b8b033.png) | ![image](https://user-images.githubusercontent.com/58642949/212329835-f58aafe8-3ea9-4270-8c4b-7a158a2e356d.png) |

- The image, title, and coding languages in the fourth project.

| Figma design | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/212330106-a23fada1-7333-4a0f-926e-f72caab5ff6b.png) | ![image](https://user-images.githubusercontent.com/58642949/212330171-165c8545-1fc7-4503-92bb-a3ee32f52466.png) |